docs/resources/Channel.md

```diff
@@ -88,7 +88,7 @@ Represents a message sent in a channel within Discord.
 |-------|------|-------------|
 | id | snowflake | id of the message |
 | channel_id | snowflake | id of the channel the message was sent in |
-| author | a [user](#DOCS_USER/user-object) object | the author of this message |
+| author | a [user](#DOCS_USER/user-object) object | the author of this message (the author of a message is not guaranteed to be a user) |
```

Comment:
[](url)-| author | a [user](#DOCS_USER/user-object) object | the author of this message |